### PR TITLE
Implements gh-pages support

### PR DESCRIPTION
**Pull request by aaronhuggins**
@headlessme @snyderj Running `npm install && npm run deploy` should build the react apps and assemple the github pages site in the `./build` directory, and then deploy the `./build` dir to a `gh-pages` branch contianing only the readme and demo site.

Sample is live at https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/

### CODE REVIEW

File: .github/workflows/collector.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: cobrowse-evidence-flow
```
Comment (headlessme) [inline .github/workflows/collector.yml:1]: Not sure this name is appropriate – copy paste error?

File: .github/workflows/collector.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    branches:
+      - master
+
+jobs:
+  gh-pages:
+    name: Github Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-user@v1
```
Comment (headlessme) [inline .github/workflows/collector.yml:13]: Is this action (fregante/setup-git-user) really needed? It doesn't seem to do much

File: README.md [hunk 3]
```diff
@@ -21,6 +30,24 @@ npm install
 npm start
 ```
 
+## Custom Agent Demo
+
+Our custom agent demo allows you to view and interact with a completely replaced agent user interface.
```
Comment (headlessme) [inline README.md:35]: Links to demos should be in each section as well.

File: README.md [hunk 4]
```diff
@@ -4,15 +4,24 @@ Full documentation available at https://docs.cobrowse.io/agent-side-integrations
...
 ## Overview
 
-Our Agent API can be used to build 100% customized agent-side integrations into your own products and services. 
+Our Agent API can be used to build 100% customized agent-side integrations into your own products and services.
 
-## Quick start
+## Live demo links
 
-Navigate to our [Agent JS API Demo Page](https://cobrowse-agent-sdk-examples.cbrws.io/) and open up our [Cobrowse.io Online Demo](https://cobrowse.io/demo) in another browser tab to generate your Demo ID. 
+- [Agent JS API Demo Page](https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/react-example/)
```
Comment (headlessme) [inline README.md:11]: Change to cobrowseio.github.io

File: README.md [hunk 5]
```diff
@@ -21,6 +30,24 @@ npm install
 npm start
 ```
 
+## Custom Agent Demo
+
+Our custom agent demo allows you to view and interact with a completely replaced agent user interface.
+
+### Quick start
+
+Navigate to our [Custom Agent Demo Page](https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/custom-agent-demo/) to automatically start an agent session and simulate the agent and customer experience.
```
Comment (headlessme) [inline README.md:39]: Change to cobrowseio.github.io

File: README.md [hunk 6]
```diff
@@ -4,15 +4,24 @@ Full documentation available at https://docs.cobrowse.io/agent-side-integrations
...
 
-Our Agent API can be used to build 100% customized agent-side integrations into your own products and services. 
+Our Agent API can be used to build 100% customized agent-side integrations into your own products and services.
 
-## Quick start
+## Live demo links
 
-Navigate to our [Agent JS API Demo Page](https://cobrowse-agent-sdk-examples.cbrws.io/) and open up our [Cobrowse.io Online Demo](https://cobrowse.io/demo) in another browser tab to generate your Demo ID. 
+- [Agent JS API Demo Page](https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/react-example/)
+- [Custom Agent Demo Page](https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/custom-agent-demo/)
```
Comment (headlessme) [inline README.md:12]: Change to cobrowseio.github.io

File: README.md [hunk 7]
```diff
@@ -4,15 +4,24 @@ Full documentation available at https://docs.cobrowse.io/agent-side-integrations
...
+
+## React Example
+
+Our React example demonstrates custom widgets to show how you can completely customize your Cobrowse.io experience.
+
+### Quick start
+
+Navigate to our [Agent JS API Demo Page](https://aaronhuggins.github.io/cobrowse-agent-sdk-examples/react-example/) and open up our [Cobrowse.io Online Demo](https://cobrowse.io/demo) in another browser tab to generate your Demo ID.
```
Comment (headlessme) [inline README.md:20]: Change to cobrowseio.github.io

File: package.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Dummy package for github pages build.",
+  "main": "\"\"",
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
+    "build": "./build.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aaronhuggins/cobrowse-agent-sdk-examples.git"
```
Comment (headlessme) [inline package.json:14]: Change to cobrowseio user in url

File: package.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/aaronhuggins/cobrowse-agent-sdk-examples/issues"
```
Comment (headlessme) [inline package.json:19]: Change to cobrowseio user in url

File: package.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "homepage": "https://github.com/aaronhuggins/cobrowse-agent-sdk-examples#readme",
```
Comment (headlessme) [inline package.json:21]: Change to cobrowseio user in url

**Review by headlessme — CHANGES_REQUESTED** [review]

File: .github/workflows/collector.yml (hunk 1 above)
Comment (aaronhuggins) [reply .github/workflows/collector.yml:1]: Ha. Yeah, I should fix that in my template file, as well as here.

File: .github/workflows/collector.yml (hunk 2 above)
Comment (aaronhuggins) [reply .github/workflows/collector.yml:13]: I'll try running it again without it.

File: package.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "git+https://github.com/cobrowseio/cobrowse-agent-sdk-examples.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/cobrowseio/cobrowse-agent-sdk-examples/issues"
+  },
+  "homepage": "https://github.com/cobrowseio/cobrowse-agent-sdk-examples#readme",
+  "devDependencies": {
+    "gh-pages": "^3.2.3"
```
Comment (headlessme) [inline package.json:23]: Is this still needed?

**Pull request merged**